ranker/ranker.py

```diff
@@ -44,8 +44,17 @@ def index():
 @app.route('/query/<query>', methods=['GET'])
 def rankQuery(query):
   log('Ranker', 'Received query: '+query)
+  index = query.find(":")
+  newStr = str(index)
+  log('Ranker', 'Number: '+ newStr)
+  if index != -1: 
+    excludedTerms = query[index+1:len(query)]
+    excludedTerms.split(" ")
+  else: 
+    excludedTerms = []
+
```

Comment:
Create a new string containing everything before the ' :' Right now, its still wasting time and processing the excluded terms and pass that into the queryTerms = stemQuery(x, stopwords)